styles.css

```diff
@@ -60,6 +60,48 @@ body {
   font: 18px/21px Roboto, arial, helvetica, sans-serif;
 }
 
+@keyframes pulse1 {
+  0% {
+    transform: scale(.9, .9);
+  }
+  50% {
+    transform: scale(1, 1); opacity: .8;
+    }
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/FirstWebsait/pullRequest?prid=6357730)